templates/p5_gfd.bt

```diff
@@ -259,6 +259,20 @@ typedef struct
  TBool32 TMaterialFlags_Flag1            : 1;
 } TMaterialFlags;
 
+typedef struct
+{ 
+    EMaterialMapsTexcoord TMaterialMaps_Filler           : 5<hidden=true>;
```

Comment:
not an enum, so the member names dont have to prefixed with TMaterialMaps

Reply:
so should it just be Filler

Reply:
yes, same goes for the rest